### ISSUE TITLE
Integrate component demos

### ISSUE THREAD

**Issue by rexxars** [vaffel/react-components#7]
A really awesome feature, which has been suggested by multiple people on IRC, is to have demos of each component available through the site.

This will require a certain amount of standardization and security considerations, obviously.

**Comment by jecel0911** [comment]
Agree, it would be awesome to have demos. webpack some config standard will be a good start.
